### DISCUSSION TITLE
how to override the default namespace kubescape-host-scanner to someother namespace

### DISCUSSION THREAD

**Discussion by jaiprasad266** [kubescape/kubescape#836] [Q&A] [answered]
Hi ,

when this cmd is executed ```kubescape scan --enable-host-scan --format json --format-version v2 --output results.json```

```kubescape``` is deploying a host scanner pod in ```kubescape-host-scanner``` namespace, due to some technical reasons I want this to be deployed in a different namespace , how can we do that ?

**Comment by Bezbran** [comment, marked as answer]
Hi @jaiprasad266, you can customize the host scanner daemonset as described [here](https://hub.armosec.io/docs/host-sensor).

I hope this will fulfill your needs.

**Reply by jaiprasad266** [reply]
Hi will try it out , thank u

**Reply by Bezbran** [reply]
OK, keep us in the loop 😄

**Reply by jaiprasad266** [reply]
I am trying with kubescape operator now , to override the namespace I have to edit the host-scanner-definition.yaml and it is not possible to override with --set

**Reply by Bezbran** [reply]
With kubescape operator it's should be even more easy, you have the `host-scanner-definition` configmap which you can edit.
